chcemvediet/adapters.py

```diff
@@ -0,0 +1,13 @@
+from allauth.account.adapter import DefaultAccountAdapter
+from django.contrib.auth import get_backends
+
+from poleno.invitations.adapters import InvitationsAdapter
+
+
+backend = get_backends()[0]
+
+class AccountAdapter(InvitationsAdapter, DefaultAccountAdapter):
```

Comment:
V `DefaultAccountAdapter` je definicia:
```
    def login(self, request, user):
        from django.contrib.auth import login
        # HACK: This is not nice. The proper Django way is to use an
        # authentication backend
        if not hasattr(user, 'backend'):
            user.backend \
                = "allauth.account.auth_backends.AuthenticationBackend"
        login(request, user)
```
Kedze tam je natvrdo pomenovany backend a my sme definovali novy, stalo sa to nefunkcne. Tak ako je pisane v komentari. Kvoli tomu nam spadol test `test_post_signup_form_with_valid_data_creates_user_and_his_profile`. Uzivatela sice vytvorilo, ale jeho `login` bol neuspesny, spravil sa nam redirect na login page a bolo potrebne sa prihlasit este raz. Inak vsetko ostatne fungovalo, nemalo to zasadny vplyv na funkcnost stranky.

Reply:
Good catch. Este, ze mame testy, inac by sme si to vsimli az na produkcii, keby sa niekomu nepodarilo zaregistrovat.

Mam vsak otazku ku:
```
backend = get_backends()[0]
```
Nezalezi na tom, ktory backend pouzijeme? `[0]` je `DjangoModelBackendWithAdminLoginAs`. Nemali by sme pouzit `AllauthAuthenticationBackendWithAdminLoginAs`?

Ak na tom zalezi, tak asi spoliehat sa na poradie backendov v `get_backends()` je dost tazke. Ak sa da, radsej by som includol settings a pouzil string z `AUTHENTICATION_BACKENDS`.

Reply:
Funguju oba nase backendy. :ok: pouzil som explicitne string s cestou.

Reply:
@viliambalaz Riadok `backend = get_backends()[0]` teraz uz asi mozes vyhodit, ci?

Reply:
:ok: